src/components/common/SearchPanels/SearcPanelSecondary.jsx

```diff
@@ -16,11 +16,13 @@ const SearchPanelSecondary = () => {
       onSubmit={handleSearch}
     >
       <input
-        className="flex-grow bg-transparent font-fourth text-xl/5 italic text-dark-color placeholder-dark-color outline-none"
+        className="flex-grow bg-transparent font-fourth text-xl/5 italic text-dark-color placeholder-dark-color/40 outline-none"
         type="search"
         name="query"
-        placeholder="India | Deli"
+        placeholder="India | Delhi"
         aria-label="Search for guides by city or country name"
+        value={query}
```

Comment:
You can try using the combination `defaultValue` + taking value from input ref

It might help to throw out the logic storing the value because we actually need it only when the user clicks submit button